terrain/voxel_lod_terrain.cpp

```diff
@@ -686,7 +696,7 @@ void VoxelLodTerrain::_process() {
 			_block_updater->pop(output);
 			_stats.updater = output.stats;
 
-			for (unsigned int i = 0; i < output.blocks.size(); ++i) {
+			for (unsigned int i = 0; i < (unsigned int)output.blocks.size(); ++i) {
```

Comment:
`i` didnt need to be unsigned

Reply:
i was originally unsigned, which produces the warning. I'll remove unsigned from both so it matches above.